modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTable.scala

```diff
@@ -269,15 +273,31 @@ private sealed trait SequenceTableBuilder[S: Eq, D <: DynamicConfig: Eq]
                 case _                                    =>
                   TagMod.empty
 
-        PrimeAutoHeightVirtualizedTable(
-          table,
-          estimateSize = _ => 25.toPx,
-          overscan = 8,
-          containerRef = resize.ref,
-          tableMod = TagMod(tableStyle),
-          rowMod = row => computeRowMods(row.original),
-          headerCellMod = computeHeaderCellMods,
-          cellMod = computeCellMods
+        React.Fragment(
+          // Render the expand all button in the ObsHeader
+          Option(dom.document.getElementById("sequence-table-expand-all")).map {
+            val (icon, label) =
+              if table.getIsAllRowsExpanded() then (Icons.Minus, "Collapse all visits")
+              else (Icons.Plus, "Expand all steps")
+            ReactPortal(
```

Comment:
This feels a bit hacky. The "portal" renders this component (the button) in a different node (from the getElementById), which is actually in the ObsHeader.scala. But I couldn't think of a clean way to get the button to render there (since you need the `table` object)